src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$white: #fff;
+$blue-accent: #00acdc;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+  font-family: "Roboto", sans-serif;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__name {
+    width: 166px;
+    margin-top: 40px;
+    margin-bottom: 0;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    align-self: flex-start;
+    margin-left: 16px;
+    margin-top: 4px;
+    color: $secondary;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__rating {
+    margin-top: 8px;
+    display: flex;
+    width: 166px;
+    flex-direction: column;
+    justify-content: flex-end;
+    color: $main-accent;
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)